backend/models/bcf.js

```diff
@@ -408,816 +464,639 @@ function getBCFMarkup(issue, account, model, unit) {
 	});
 }
 
-bcf.getBCFZipReadStream = function(account, model, username, branch, revId, ids, useIssueNumbers = false) {
-
+bcf.getBCFZipReadStream = function(account, model, issues, unit) {
 	const zip = archiver.create("zip");
 
 	// Optional project extensions not currently utilised
 	// zip.append(new Buffer.from(getProjectExtension(model), "utf8"), {name: "project.bcfp"})
 	zip.append(new Buffer.from(getBCFVersion(), "utf8"), {name: "bcf.version"});
 
-	const projection = {};
-	const noClean = true;
-	let settings;
-
-	return ModelSetting.findById({account, model}, model).then(_settings => {
-
-		settings = _settings;
-		return Issue.findByModelName(account, model, branch, revId, projection, ids, noClean, useIssueNumbers);
+	const bcfPromises = [];
 
-	}).then(issues => {
+	issues.forEach(issue => {
 
-		const bcfPromises = [];
+		const issueAccount = (issue.origin_account) ? issue.origin_account : account;
+		const issueModel = (issue.origin_model) ? issue.origin_model : model;
 
-		issues.forEach(issue => {
+		bcfPromises.push(
+			// FIXME
```

Comment:
what is this fixme for?

Reply:
I can't remember, but this function seems much clearer when compared to the import, so I think I'll remove the FIXME.